proto/errors.proto

```diff
@@ -26,7 +26,7 @@ import "gogoproto/gogo.proto";
 // A NotLeaderError indicates that the current range is not the
 // leader. If the leader is known, its Replica is set in the error.
 message NotLeaderError {
-  optional Replica leader = 1 [(gogoproto.nullable) = false];
+  optional Replica leader = 1;
```

Comment:
can you add a TODO for me here? I don't remember it now but there was some place in one of my PRs I wanted to update when this lands.